### PR TITLE
Match the way WordPress Core locates `wp-config.php`

### PR DESCRIPTION
**Pull request by VladimirAus**
Fix for #4856 Updated locate_wp_config() to match core.

### CODE REVIEW

**Comment by swissspidy** [comment]
Why the error silencing?

**Comment by VladimirAus** [comment]
@swissspidy You need to be more specific. Who is silencing and which error?

File: php/utils.php [hunk 1]
```diff
@@ -234,8 +239,8 @@ function locate_wp_config() {
 
 		if ( file_exists( ABSPATH . 'wp-config.php' ) ) {
 			$path = ABSPATH . 'wp-config.php';
-		} elseif ( file_exists( ABSPATH . '../wp-config.php' ) && ! file_exists( ABSPATH . '/../wp-settings.php' ) ) {
-			$path = ABSPATH . '../wp-config.php';
+		} elseif ( @file_exists( dirname( ABSPATH ) . '/wp-config.php' ) && ! @file_exists( dirname( ABSPATH ) . '/wp-settings.php' ) ) {
```
Comment (swissspidy) [inline php/utils.php]: Why the error silencing here? `@` silences errors in PHP, see http://php.net/manual/en/language.operators.errorcontrol.php.

**Comment by swissspidy** [comment]
@VladimirAus My bad, I thought I had added this as an inline comment. Refined the question now.

File: php/utils.php (hunk 1 above)
Comment (VladimirAus) [reply php/utils.php]: Yep, that's exactly what core is doing. 
Happy to do normal call.

Comment (schlessera) [reply php/utils.php]: This is probably to suppress the warning that can be generated:
image 2018-10-29 at 9 17 12 am

We don't need to suppress these for WP-CLI, we're not generating HTML here, so I'd prefer to get rid of the error silencing.

**Comment by schlessera** [comment]
Thanks for the PR, @VladimirAus !

**Pull request merged**